src/IB/IBExplicitHierarchyIntegrator.cpp

```diff
@@ -511,7 +511,10 @@ IBExplicitHierarchyIntegrator::setMarkers(const EigenAlignedVector<IBTK::Point>&
     IBTK::Vector v;
     v.fill(0.0);
     std::fill(velocities.begin(), velocities.end(), v);
+    // avoid having two markers at once
+    d_markers.setNull();
```

Comment:
If you reset the value of `d_markers` on the next line, do you need to call `setNull()` here?

Reply:
Yes. We need to delete the current one to remove it from the restart database before adding a new one - otherwise we will have two objects with the same name.

Reply:
Ah --- can you add that to the comment?